ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Overview/WorkloadCvesOverviewPage.tsx

```diff
@@ -45,7 +58,7 @@ function WorkloadCvesOverviewPage() {
     const { data: countsData = { imageCount: 0, imageCVECount: 0, deploymentCount: 0 }, loading } =
         useQuery(entityTypeCountsQuery, {
             variables: {
-                query: getCveStatusScopedQueryString(querySearchFilter),
+                query: getCveStatusScopedQueryString(querySearchFilter, currentCveStatus),
```

Comment:
Here is an pro and con question for mental slow cooker.

Where is boundary in component architecture to translate between:
* backend key or value (and map to human text only at rendering)
* human text (especially considering possible globalization)

To look at it another way, which way to map values:

```ts
// Map from the CVE status tab to the backend equivalent search value
const vulnerabilitySearchStateForCveStatus: Record<CveStatusTab, VulnerabilityState> = {
    Observed: 'OBSERVED',
    Deferred: 'DEFERRED',
    'False Positive': 'FALSE_POSITIVE',
} as const;
```

As it relates to query string in page address:
* currently `cveStatus=False%20Positive` and `eventKey="False Positive"`
* hypothetically `vulnerabilityStatus=FALSE_POSITIVE` and `eventKey="FALSE_POSITIVE"`

Here are a few examples of backend keys or values that I found:
* query: https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/cypress/integration/vulnmanagement/riskAcceptance/observedCVEs.test.js#L63-L83
* query string: `/main/risk?s[Vulnerability%20State]=OBSERVED`

Reply:
This is a good point, from what I can tell the more-human-readable version (e.g. "False Positive") is only used as the key for the tabs, and is translated to the backend key before inclusion into the search query. We already are using a human readable text directly in the Tab `children` so the extra level of indirection here isn't necessary.

We would only target the Tab `children` for globalization in the future anyway, so it makes sense to keep the internal tab key closer to the server value it represents. I'll take a look at this more closely and see if there are any places that disagree with my assessment above.

Reply:
An extra +1 since the same `cveStatus` naming is used in the sibling folder for reporting, but there it refers to Fixable/Not Fixable instead of vulnerability state.